src/components/application_manager/src/commands/hmi/basic_communication_on_awake_sdl.cc

```diff
@@ -29,3 +29,45 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+
+#include "application_manager/commands/hmi/basic_communication_on_awake_sdl.h"
+#include "application_manager/message_helper.h"
+#include "application_manager/resumption/resume_ctrl.h"
+
+namespace application_manager {
+
+namespace commands {
+
+OnAwakeSDLNotification::OnAwakeSDLNotification(
+    const MessageSharedPtr& message, ApplicationManager& application_manager)
+    : NotificationFromHMI(message, application_manager) {}
+
+OnAwakeSDLNotification::~OnAwakeSDLNotification() {}
+
+void OnAwakeSDLNotification::Run() {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  if (application_manager_.resume_controller().is_suspended()) {
+    {
```

Comment:
@dev-gh updated in https://github.com/smartdevicelink/sdl_core/pull/1604/commits/f6be867a635dfbc41c7d959e462b8101e7f6d164